send-transaction-service/src/send_transaction_service.rs

```diff
@@ -135,199 +139,6 @@ impl Default for Config {
 /// processing the transactions that need to be retried.
 pub const MAX_RETRY_SLEEP_MS: u64 = 1000;
 
-/// The leader info refresh rate.
-pub const LEADER_INFO_REFRESH_RATE_MS: u64 = 1000;
-
-/// A struct responsible for holding up-to-date leader information
-/// used for sending transactions.
-pub struct CurrentLeaderInfo<T>
```

Comment:
this and any other moved public interface needs to be reexported here, with deprecation annotations. even though the annotations don't actually work on re-exports, they give us documentation and something to grep for

Reply:
thanks! can you annotate with the [rust attribute](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute) instead of a comment?